packages/core/utils/create-project.ts

```diff
@@ -73,10 +56,15 @@ export async function createProject(cwd: string) {
         directory = relativePath;
     }
 
-    const projectType = await selectPrompt("Which project type do you want to create?", "kit", [
-        { label: "SvelteKit", value: "kit" },
-        { label: "Svelte", value: "svelte" },
-    ]);
+    const availableProjectTypes: PromptOption<string>[] = [];
+    if (supportKit) availableProjectTypes.push({ label: "SvelteKit", value: "kit" });
+    if (supportSvelte) availableProjectTypes.push({ label: "Svelte", value: "svelte" });
+
+    let projectType: string;
+
+    if (availableProjectTypes.length == 0) throw new Error("Failed to identify possible project types");
+    else if (availableProjectTypes.length == 1) projectType = availableProjectTypes[0].value;
```

Comment:
_the most minor of nits:_ `else` isn't necessary
```suggestion
    if (availableProjectTypes.length == 1) projectType = availableProjectTypes[0].value;
```